robot/control/Src/modules/RadioModule.cpp

```diff
@@ -0,0 +1,61 @@
+#include "modules/RadioModule.hpp"
+#include "iodefs.h"
+
+RadioModule::RadioModule(BatteryVoltage *const batteryVoltage,
+                         FPGAStatus *const fpgaStatus,
+                         KickerInfo *const kickerInfo,
+                         RobotID *const robotID,
+                         KickerCommand *const kickerCommand,
+                         MotionCommand *const motionCommand,
+                         RadioError *const radioError)
+    : batteryVoltage(batteryVoltage), fpgaStatus(fpgaStatus),
+      kickerInfo(kickerInfo), robotID(robotID),
+      kickerCommand(kickerCommand), motionCommand(motionCommand),
+      radioError(radioError), link(),
+      secondRadioCS(RADIO_R1_CS) {
+
+    secondRadioCS = 1;
```

Comment:
One more comment I forgot to make. Can we _please_ just use `.write()` instead of `operator=` for setting DIO values? This is so confusing - I had to read this code several times before I realized that `secondRadioCS` isn't an integer.

I also put an issue in mtrain-firmware to remove this API. IMO it does nothing but create confusion.